src/java.naming/share/classes/module-info.java

```diff
@@ -79,11 +79,16 @@
  * <ul>
  *     <li>{@systemProperty com.sun.jndi.ldap.object.trustSerialData}:
  *          <br>The value of this system property is the string representation of a boolean value
- *          which allows to control the deserialization of java objects from the 'javaSerializedData'
- *          LDAP attribute. To prevent the deserialization of java objects from the 'javaSerializedData'
- *          attribute, the system property value can be set to 'false'.
- *          <br>If the property is not specified then the deserialization of java objects
- *          from the 'javaSerializedData' attribute is allowed.
+ *          which allows to control the deserialization of java objects from the {@code javaSerializedData} LDAP
```

Comment:
"allows to control"? Maybe this should be "that controls".

Is there anywhere that "javaRemoveLocation", and "javaReferenceAddress" could link to?

Reply:
`javaReferenceAddress` is defined in RFC 2713, section 3.6 - but `javaRemoteLocation` is not - it's marked deprecated in our code base so maybe it came from some early draft of this representation.

Reply:
> `javaReferenceAddress` is defined in RFC 2713, section 3.6 - but `javaRemoteLocation` is not - it's marked deprecated in our code base so maybe it came from some early draft of this representation.

Correct, and since these attributes are not documented it would be great to at least mention them in `java.naming` module javadoc.

> "allows to control"? Maybe this should be "that controls".

Thanks, will change that